GOVERNANCE.md

```diff
@@ -4,7 +4,7 @@ This document describes the rules and governance of the project. It is meant to
 
 * **Team members**: Any members of the private [grafana-team][team] Google group.
 
-* **Maintainers**: Maintainers lead an individual project or parts thereof ([`MAINTAINERS.md`][maintainers.md]).
+* **Maintainers**: Maintainers lead an individual project or parts thereof ([`MAINTAINERS.md`][MAINTAINERS.md]).
```

Comment:
```suggestion
* **Maintainers**: Maintainers lead an individual project or parts thereof ([`MAINTAINERS.md`](https://github.com/grafana/grafana/blob/master/MAINTAINERS.md)).
```

Reply:
the link points to [maintainers.md]: https://github.com/search?l=&q=org%3Agrafana+filename%3AMAINTAINERS.md+path%3A%2F&ref=advsearch&type=Code&utf8=%E2%9C%93
198